### ISSUE TITLE
Add accessibility options in pdf reader [enhancement]

### ISSUE THREAD

**Issue by fraterlinux** [JakubMelka/PDF4QT#108]
A universal FOSS pdf reader for everyone should have accessibility options for all types of readers.

A list of accessibility options to be implemented:

1) PDF color inversion,
2) colors to shades of gray,
3) high contrast that highlights colors,
4) option to change background and text color, including for scanned PDF

Example background changed to black and text to green in a scanned pdf:
https://ibb.co/smsGbhc

**Comment by JakubMelka** [comment]
Hello @fraterlinux , PDF color inversion should work - I have implemented it before.

**Comment by fraterlinux** [comment]
> Hello @fraterlinux , PDF color inversion should work - I have implemented it before.

I can't find a simple way to change the colors. Could you make accessibility options simpler and easier?

**Comment by JakubMelka** [comment]
Well, it is in the menu:
![image](https://github.com/JakubMelka/PDF4QT/assets/11367048/7ecd71a1-4af7-463a-9216-3faee2059e81)

**Comment by JakubMelka** [comment]
But I will implement shades of gray, high constract and change background and text color.

**Comment by fraterlinux** [comment]
> But I will implement shades of gray, high constract and change background and text color.

Thank you very much!!!

**Comment by JakubMelka** [comment]
@fraterlinux, I have implemented it. New modes include:

- Inverted Colors
- High Contrast Colors
- Grayscale Colors
- Bitonal (convert to pure black/white without gray areas)
- Custom colors for background / text
